tests/runner/test_dataset_block.py

```diff
@@ -103,6 +127,113 @@ def test_partial_block_for_chunked_data(slicing_dim: Literal[0, 1, 2], last_in_c
     assert block.slicing_dim == slicing_dim
 
 
+@pytest.mark.parametrize("slicing_dim", [0, 1, 2])
+def test_partial_block_for_chunked_data_with_padding_center(
+    slicing_dim: Literal[0, 1, 2]
+):
+    block_shape = [10, 10, 10]
+    block_shape[slicing_dim] = 6
+    start_index = 3
+    data = np.ones(block_shape, dtype=np.float32)
+    global_index = [0, 0, 0]
+    global_index[slicing_dim] = 10 + start_index
```

Comment:
Is this 10 connected to the 10 on line 152 where there is `chunk_start=10`? (If it is, would be good to put it in a var for reuse+understanding, but if not then no worries)